quick-start/e2e/specs/mappings/typeAhead.ts

```diff
@@ -140,7 +140,7 @@ export default function() {
         mappingsPage.saveMapButton().click();
         browser.wait(EC.elementToBeClickable(mappingsPage.entityMapping('MapTypeAhead')));
         //flicker bug, sleep will be removed once it's fixed
```

Comment:
an actual bug in another project?  if so a link will help us maintain in the future

Reply:
The mapping page is flickering/reloading. Filed a bug: https://project.marklogic.com/jira/browse/DHFPROD-1161

Reply:
I think this is a result of how the maps service is tied to entities changing, reloading the model. We should fix this.